cpp/src/gandiva/function_registry_datetime.cc

```diff
@@ -36,6 +36,7 @@ namespace gandiva {
 
 std::vector<NativeFunction> GetDateTimeFunctionRegistry() {
   static std::vector<NativeFunction> date_time_fn_registry_ = {
+      UNARY_SAFE_NULL_NEVER_BOOL(isnull, {}, day_time_interval),
```

Comment:
can you add isnull to all the types?

Reply:
@projjal suggestion applied 👍🏻

Reply:
@projjal I added `ISNULL` for `day_time_interval` and `month_interval` because the rest already has it implemented.